pytest_fixtures/user_fixtures.py

```diff
@@ -20,3 +25,33 @@ def default_viewer_role(module_org, default_location):
     ).create()
     custom_user.password = custom_password
     return custom_user
+
+
+@pytest.fixture(scope='module')
+def module_user(request, module_org, module_loc):
+    """Creates admin user with default org set to module org and shares that
+    user for all tests in the same test module. User's login contains test
+    module name as a prefix.
+
+    :rtype: :class:`nailgun.entities.Organization`
+    """
+    # take only "module" from "tests.ui.test_module"
+    test_module_name = request.module.__name__.split('.')[-1].split('_', 1)[-1]
+    login = f'{test_module_name}_{gen_string("alphanumeric")}'
```

Comment:
what do you think the risk is if API, CLI, and UI tests potentially run on the same satellite?

Reply:
I don't see any, the name includes a randomized component.  The user is also deleted once the module is done.

There is a `module_user` fixture also defined in `pytest_fixtures/api_fixtures.py`, but it isn't customized at all and simply creates a user under org/location scoped for the module as well.

Some test modules are also defining their own `module_user` fixtures.